harness/determined/cli/task.py

```diff
@@ -59,8 +59,9 @@ def agent_info(t: Dict[str, Any]) -> Union[str, List[str]]:
 
 @authentication.required
 def list_tasks(args: Namespace) -> None:
-    r = api.get(args.master, "tasks")
-    tasks = r.json()
```

Comment:
suggestion: use the python bindings that will get generated for you when you run 

`make -C proto/ clean fmt build && make -C bindings clean build`

```
def get_GetTasks(
    session: "api.Session",
) -> "v1GetTasksResponse":
    _params = None
    _resp = session._do_request(
        method="GET",
        path="/api/v1/tasks",
        params=_params,
        json=None,
        data=None,
        headers=None,
        timeout=None,
        stream=False,
    )
    if _resp.status_code == 200:
        return v1GetTasksResponse.from_json(_resp.json())
    raise APIHttpError("get_GetTasks", _resp)

```

Reply:
Done. Thanks!